content/docs/command-reference/fetch.md

```diff
@@ -194,8 +196,8 @@ Note that the `.dvc/cache` directory was created and populated.
 > for more info.
 
 Used without arguments (as above), `dvc fetch` downloads all assets needed by
-all DVC-files in the current branch, including for directories. The hash values
-`3863d0e317dee0a55c4e59d2ec0eef33` and `42c7025fc0edeb174069280d17add2d4`
+all `.dvc` files in the current branch, including for directories. The hash
```

Comment:
`all dvc.yaml and .dvc files in`...

But let's also link both terms in this case, as this Example section doesn't seem to have the link already.